src/App.jsx

```diff
@@ -1,33 +1,57 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function getVisibleMovies(movies, { filterField }) {
+  let initialMovies = movies;
+
+  if (filterField) {
+    initialMovies = initialMovies
+      .filter(movie => movie.title.toLowerCase()
+        .includes(filterField.trim().toLowerCase())
+      || movie.description.toLowerCase()
+        .includes(filterField.trim().toLowerCase()));
+  }
+
+  return initialMovies;
+}
+
+export const App = () => {
+  const [filterField, setFilterField] = useState('');
+
+  const visibleMovies = getVisibleMovies(moviesFromServer, { filterField });
+
+  return (
+    <div className="page">
+      <div className="page-content">
+        <div className="box">
+          <div className="field">
+            <label htmlFor="search-query" className="label">
+              Search movie
+            </label>
+
+            <div className="control">
+              <input
+                value={filterField}
+                type="text"
+                id="search-query"
+                className="input"
+                placeholder="Type search word"
+                onChange={(event) => {
+                  setFilterField(event.currentTarget.value);
+                }}
```

Comment:
In `React` it is better to use `event.target`

[Difference between target and currentTarget](https://stackoverflow.com/questions/10086427/what-is-the-exact-difference-between-currenttarget-property-and-target-property)

```suggestion
                onChange={(event) => setFilterField(event.target.value)}
```